### PR TITLE
Fix warnings, remove unused code, improve Makefile

### PR DESCRIPTION
**Pull request by adriweb**
Tested successfully with a normal .lua script, but nothing else (a few test files would be useful, I guess)

### CODE REVIEW

**Comment by Vogtinator** [comment]
We could simplify ```PREFIX``` to be always ```/usr/local/bin```. That should work for all platforms (except windows...)

**Comment by adriweb** [comment]
Alright, I'll do that.

**Comment by Legimet** [comment]
You can also define NOCRYPT and remove crypt.h.
And fix the misleading indentation warning:
```
minizip-1.1/zip.c: In function ‘zip64local_SearchCentralDir’:
minizip-1.1/zip.c:531:5: warning: this ‘for’ clause does not guard... [-Wmisleading-indentation]
     for (i=(int)uReadSize-3; (i--)>0;)
     ^~~
minizip-1.1/zip.c:539:7: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘for’
       if (uPosFound!=0)
```

File: README.md [hunk 1]
```diff
@@ -1,49 +1,41 @@
...
-information.
+This product is based on a derived version of MiniZip.  
+See OpenSSL-License.txt and minizip-1.1/MiniZip64\_info.txt for more information.
 
 Building it yourself
 ====================
 
 You need the OpenSSL (libssl-dev/openssl-devel) and zlib (zlib1g-dev/zlib-devel)
-development libraries. Then you can just run `make`.
+development libraries. If on macOS, you can use [brew](http://brew.sh/): `brew install openssl zlib`
```
Comment (Vogtinator) [inline README.md]: s/If on/On/

File: README.md [hunk 2]
```diff
@@ -1,49 +1,41 @@
-Luna v1.0.1 for TI-Nspire
+Luna v1.0.1
```
Comment (Vogtinator) [inline README.md]: Luna 1.0.1

File: README.md [hunk 3]
```diff
@@ -1,49 +1,41 @@
...
 
-Multiple files:            `luna Document.xml Problem1.xml [Problem2.xml...] OUTFILE.tns`
-
-If the input is `'-'`, it reads the file from the standard input.
-Make sure to encode your Lua or Problem file in UTF-8 if it contains special
-characters. You can also pack arbitrary files like images into the TNS.
-
-Contact
+Bugs, Feedback...
 =======
```
Comment (Vogtinator) [inline README.md]: Align to title length

File: README.md [hunk 4]
```diff
@@ -1,49 +1,41 @@
-Luna v1.0.1 for TI-Nspire
+Luna v1.0.1
 =======================
```
Comment (Vogtinator) [inline README.md]: Align to title length

File: README.md [hunk 5]
```diff
@@ -1,52 +1,39 @@
...
-See OpenSSL-License.txt and minizip-1.1/MiniZip64\_info.txt for more
-information.
+This product is based on a derived version of MiniZip.  
+See OpenSSL-License.txt and minizip-1.1/MiniZip64\_info.txt for more information.
 
-Building it yourself
-====================
+## Building it yourself
 
 You need the OpenSSL (libssl-dev/openssl-devel) and zlib (zlib1g-dev/zlib-devel)
```
Comment (Vogtinator) [inline README.md]: s/openssl-devel/libopenssl-devel/

File: README.md [hunk 6]
```diff
@@ -1,52 +1,39 @@
...
-
-License
-=======
+## License
 
 Luna is covered by the Mozilla Public License v1.1.
 
-This product includes software developed by the OpenSSL Project for use in the
-OpenSSL Toolkit (https://www.openssl.org/).
+This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit (https://www.openssl.org/).
```
Comment (Vogtinator) [inline README.md]: Does that still apply? I guess it references the now gone libeay32.dll

File: README.md [hunk 7]
```diff
@@ -1,52 +1,35 @@
...
-=======
+## License
 
 Luna is covered by the Mozilla Public License v1.1.
+It is based on a derived version of MiniZip. See minizip-1.1/MiniZip64_info.txt for more information.
 
-This product includes software developed by the OpenSSL Project for use in the
-OpenSSL Toolkit (https://www.openssl.org/).
-
-This product is based on a derived version of MiniZip.
```
Comment (Vogtinator) [inline README.md]: Keep the minizip information

Comment (Vogtinator) [inline README.md]: Ignore that, github's diff viewer does not show moves...

**Pull request merged**